src/TouristInfoCards.js

```diff
@@ -0,0 +1,42 @@
+import React from "react";
+
+const city = [
```

Comment:
Small thing but up to you it may be better to rename this to 'cities' as it is an array of cities.